mmv1/products/vertexai/api.yaml

```diff
@@ -103,6 +103,238 @@ objects:
+          - !ruby/object:Api::Type::String
+            name: 'serviceAccount'
+            description: |
+              The service account that the DeployedModel's container runs as. Specify the email address of the service account. If this service account is not specified, the container runs as a service account that doesn't have access to the resource project.  
+              <br>
+              Users deploying the Model must have the iam.serviceAccounts.actAs permission on this service account.
+          - !ruby/object:Api::Type::Boolean
+            name: 'enableContainerLogging'
+            description: |
+              If true, the container of the DeployedModel instances will send stderr and stdout streams to Stackdriver Logging.  
+              Only supported for custom-trained Models and AutoML Tabular Models.
+          - !ruby/object:Api::Type::String
+            name: 'enableAccessLogging'
+            description: |
+              These logs are like standard server access logs, containing information like timestamp and latency for each prediction request.  
+              Note that Stackdriver logs may incur a cost, especially if your project receives prediction requests at a high queries per second rate (QPS). Estimate your costs before enabling this option.
+          - !ruby/object:Api::Type::NestedObject
+            name: 'privateEndpoints'
+            input: false
+            output: true
+            description: |
+              Provide paths for users to send predict/explain/health requests directly to the deployed model services running on Cloud via private services access. This field is populated if network is configured.  
+            properties:  
+              - !ruby/object:Api::Type::String
+                name: 'predictHttpUri'
+                description: Http(s) path to send prediction requests.
+                input: false
+                output: true
+              - !ruby/object:Api::Type::String
+                name: 'explainHttpUri'
+                description: Http(s) path to send explain requests.
+                output: true
+                input: false
+              - !ruby/object:Api::Type::String
+                name: 'healthHttpUri'
+                description: Http(s) path to send health requests.
+                input: false
+                output: true
+          - !ruby/object:Api::Type::NestedObject
+            name: 'dedicatedResources'
+            description: |
+              A description of resources that are dedicated to the DeployedModel, and that need a higher degree of manual configuration.  
+              Cannot be used with `automaticResources`
+            properties:
+              - !ruby/object:Api::Type::NestedObject
+                name: 'MachineSpec'
+                required: true
+                description: |
+                  Immutable. The specification of a single machine used by the prediction.
```

Comment:
`Immutable` in these docs tend to not be updatable, which corresponds to `input: true`. Did you find that wasn't the case? The cloud docs are wrong sometimes!